Tests/SonarScanner.MSBuild.TFS.Test/Classic/BinaryToXmlCoverageReportConverterTests.cs

```diff
@@ -210,7 +210,7 @@ public void Initialize_CanGetGetExeToolPathFromEnvironmentVariable_FullPathToCod
             var logger = new TestLogger();
             var config = new AnalysisConfig();
             var filePath = Path.Combine(Environment.CurrentDirectory, "CodeCoverage.exe");
-            File.Create(filePath);
+            using var f = new TestFile(filePath);
```

Comment:
Is this needed? The variable is not used and the test passes even if I delete this line.

Reply:
The tests pass sometimes. There are two problems solved by "TestFile":
* `File.Create` returns a `Stream` which must be closed before the file can be deleted
* A test should delete the file after it is run.
The `var f` is not needed but the `using` is as it ensures that the file gets deleted in "finally".
Unfortunately, there are no good patterns for `using` where you are not interested in the variable but need to dispose of it at the end:
```cs
        using var _ = new FileStream("", FileMode.Append); // Here StyleCop complains about the name
        
        using (new FileStream("", FileMode.Append)) // Here you need an extra level of nesting
        {
        }
```